core/src/main/java/com/homeaway/streamplatform/streamregistry/UUIDMDCRequestFilter.java

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2018 Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamplatform.streamregistry;
+
+import java.util.UUID;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+
+import org.slf4j.MDC;
+
+public class UUIDMDCRequestFilter implements ContainerRequestFilter {
```

Comment:
javadoc?

Reply:
fixed and addressed at https://github.com/homeaway/stream-registry/pull/133